nequip/train/trainer_wandb.py

```diff
@@ -1,31 +1,23 @@
-""" Nequip.train.trainer
-
-Todo:
-
-isolate the loss function from the training procedure
-enable wandb resume
-make an interface with ray
-
-"""
-
 import wandb
 
 from .trainer import Trainer
 
 
 class TrainerWandB(Trainer):
-    """Class to train a model to minimize forces"""
-
-    def __init__(self, **kwargs):
-        Trainer.__init__(self, **kwargs)
+    """Trainer class that adds WandB features"""
 
     def end_of_epoch_log(self):
         Trainer.end_of_epoch_log(self)
         wandb.log(self.mae_dict)
 
-    def init_model(self):
+    def init(self):
+        super().init()
```

Comment:
```suggestion
        _initialized = self._initialized
        super().init()
```

Reply:
```
def init(self):
        """initialize optimizer"""
        if self.model is None:
            return

        self.model.to(self.torch_device)
        self.init_objects()

        self._initialized = True
```
no? not if model is none which is what I'm guarding against

Reply:
ah. you are right. I was confused.